### ISSUE TITLE
Txt2AL doesn't create OnAction trigger in PageExt

### ISSUE THREAD

**Issue by alapenas** [microsoft/AL#6035]
Tested with Txt2AL version:
BC14 cu13 BE

Txt2AL missed to convert OnAction trigger code to AL PageExtension.
In my project, there were more missed actions in List pages, but I couldn't find the pattern or reason why.

**Delta file:**
```
OBJECT Modification "Sales Lines Subform"(Page 7172)
{
  OBJECT-PROPERTIES
  {
    Date=20200108D;
    Time=120000T;
    Version List=NAVW110.00,GS;
  }
  PROPERTIES
  {
    Target="Sales Lines Subform"(Page 7172);
    TargetPageType=ListPart;
  }
  CHANGES
  {
    { Insertion         ;ChangedElements=ActionCollection
    {
      { 1100096043;  ;ActionContainer;
                      Name=ActionContainer1100096043;
                      ActionContainerType=ActionItems }
      { 1100096046;1 ;Action    ;
                      Name=A_CopyToDoc;
                      CaptionML=[ENU=Cop&y to Document;
                                 FRB=Cop&ier dans le document;
                                 NLB=&Kopiëren naar document];
                      Visible=GBo_CopyToDocVisible;
                      Image=CopyDocument;
                      OnAction=BEGIN
                                 F_CopyLinesToDocLocal;
                                 // you cannot close main page
                                 //CurrPage.CLOSE;
                               END;
                                }
      { 1100096044;1 ;Action    ;
                      Name=A_TEST;
                      Visible=FALSE;
                      OnAction=BEGIN
                                 ShowMessage;
                               END;
                                }
    }
     }
  }
  CODE
  {

    BEGIN
    END.
  }
}
```

**Result:**
```
pageextension 70000001 pageextension70000001 extends "Sales Lines Subform"
{
    actions
    {
        addfirst(processing)
        {
            action(A_CopyToDoc)
            {
                Caption = 'Cop&y to Document';
                Image = CopyDocument;
                Visible = GBo_CopyToDocVisible;
            }
            action(A_TEST)
            {
                Visible = false;
            }
        }
    }
}
```

**It should be:**
```
pageextension 70000001 pageextension70000001 extends "Sales Lines Subform"
{
    actions
    {
        addfirst(processing)
        {
            action(A_CopyToDoc)
            {
                Caption = 'Cop&y to Document';
                Image = CopyDocument;
                Visible = GBo_CopyToDocVisible;

                trigger OnAction()
                begin
                    F_CopyLinesToDocLocal;
                    // you cannot close main page
                    //CurrPage.CLOSE;
                end;
            }
            action(A_TEST)
            {
                Visible = false;

                trigger OnAction()
                begin
                    ShowMessage;
                end;
            }
        }
    }
}
```

**Comment by qutreson** [comment]
Which version of Business Central are you using when running the txt2al?

**Comment by alapenas** [comment]
I used Txt2Al from BC14 cu13 BE and ran it with CMD.

**Comment by lv-janpieter** [comment]
We had a simular problem on page 1230 SEPA Direct Debit Mandates. What page 7172 Sales Lines Subform and 1230 have in common is that both do not have actions in standard BC. 

To thest the theory I tried adding a dummy action inside the standard BC object. Also I added the same dummy action in our custom solution so the delta remains the same. After doing that the onaction was converted.

BC 14.15 build 43793 (CU14)
